Userland/sleep.cpp

```diff
@@ -42,22 +74,46 @@ int main(int argc, char** argv)
     }
 
     if (argc != 2) {
-        printf("usage: sleep <seconds>\n");
+        printf("Usage:\n        sleep <seconds>\n");
+        printf("\nArguments:\n        seconds Fractional amount of seconds, e.g. 3, 0.01, 1337.42, or inf. The number will be rounded down to the nearest possible amount (usually by 1 ms).\n");
```

Comment:
\<sergey\>Please use `Core::ArgsParser`:smile:\</sergey\>

Reply:
I wasn't sure whether it's overkill – looking at that string literal again, now I think `ArgsParser` is a better idea indeed!

Reply:
Has been done in 62283ade912bbe0c2ff92509a0ed2a787f330853 - no longer relevant.